src/yuzu/main.cpp

```diff
@@ -2869,18 +2871,13 @@ void GMainWindow::OnGameListCreateShortcut(u64 program_id, const std::string& ga
 #endif // __linux__
 
     std::filesystem::path target_directory{};
-    // Determine target directory for shortcut
-#if defined(WIN32)
-    const char* home = std::getenv("USERPROFILE");
-#else
-    const char* home = std::getenv("HOME");
-#endif
-    const std::filesystem::path home_path = (home == nullptr ? "~" : home);
-    const char* xdg_data_home = std::getenv("XDG_DATA_HOME");
 
+    // Determine target directory for shortcut
     if (target == GameListShortcutTarget::Desktop) {
-        target_directory = home_path / "Desktop";
-        if (!Common::FS::IsDir(target_directory)) {
+        QString desktopPath = QStandardPaths::writableLocation(QStandardPaths::DesktopLocation);
+        target_directory = desktopPath.toUtf8().toStdString();
+        QDir dir(QString::fromStdString(target_directory.generic_string()));
```

Comment:
```suggestion
        const QDir dir(QString::fromStdString(target_directory.generic_string()));
```
also why are we casting from qstring to std::string, then back to qstring?

Reply:
I was having problems mainly with a 'Á' character on my desktop "F:\Area de Trabalho", as you suggest it should be?